packages/cli/contracts/src/codegen/tables/Dynamics1.sol

```diff
@@ -1495,37 +1495,67 @@ library Dynamics1 {
 }
 
 function toStaticArray_bytes32_1(bytes32[] memory _value) pure returns (bytes32[1] memory _result) {
-  // in memory static arrays are just dynamic arrays without the length byte
-  assembly {
-    _result := add(_value, 0x20)
+  if (_value.length < 1) {
+    // ignore invalid dynamic arrays that are too small
+    return _result;
```

Comment:
if this is invalid, should this revert instead of returning?

Reply:
a 0-length array is kinda valid (it's uninitialized), so maybe I should change the comment?
and other cases are weird and shouldn't happen, but it's cheaper to return empty than add more checks